views/homepage.handlebars

```diff
@@ -1,111 +1,47 @@
-            Trip - 5 comments</a>
-          <span class="list-item"><strong>Posted by:</strong>
-            Kristin Gaito</span>
-          <span class="list-item"><strong>Description:</strong>
-            Tour of all of the english pubs</span>
-          <span class="list-item"><strong>Location:</strong>
-            London, England</span>
-          <span class="list-item"><strong>Start Date:</strong> 2020-08-15</span>
-          <span class="list-item"><strong>End Date:</strong> 2020-08-22</span>
-          <a href="/single-blog/2">More Details</a>
-        </div>
-      </article>
-    </li>
-    <li>
-      <article class="post">
-        <div class="post-card">
-          <a href="/single-blog/2" class="blog-title" target="_blank">A Paris
-            Adventure - 35 comments</a>
-          <span class="list-item"><strong>Posted by:</strong>
-            Christine Baxter</span>
-          <span class="list-item"><strong>Description:</strong>
-            A trip filled with art and pastries</span>
-          <span class="list-item"><strong>Location:</strong>
-            Paris, France</span>
-          <span class="list-item"><strong>Start Date:</strong> 2020-08-15</span>
-          <span class="list-item"><strong>End Date:</strong> 2020-08-22</span>
-          <a href="/single-blog/2">More Details</a>
-        </div>
-      </article>
-    </li>
-    <li>
-      <article class="post">
-        <div class="post-card">
-          <a href="/single-blog/2" class="blog-title" target="_blank">Family
-            trip to the City - 23 comments</a>
-          <span class="list-item"><strong>Posted by:</strong> Hank Kapka</span>
-          <span class="list-item"><strong>Description:</strong>
-            Sightseeing visit to Central Park, the MET, the Statue of Liberty,
-            and Times Square</span>
-          <span class="list-item"><strong>Location:</strong>
-            New York, USA</span>
-          <span class="list-item"><strong>Start Date:</strong> 2020-08-15</span>
-          <span class="list-item"><strong>End Date:</strong> 2020-08-22</span>
-          <a href="/single-blog/2">More Details</a>
-        </div>
-      </article>
-    </li>
-    <li>
-      <article class="post">
-        <div class="post-card">
-          <a href="/single-blog/2" class="blog-title" target="_blank">Trip to
-            Down Under - 47 comments</a>
-          <span class="list-item"><strong>Posted by:</strong> Raman TV</span>
-          <span class="list-item"><strong>Description:</strong>
-            Harbor views and museums</span>
-          <span class="list-item"><strong>Location:</strong>
-            Sydney, Australia</span>
-          <span class="list-item"><strong>Start Date:</strong> 2020-08-15</span>
-          <span class="list-item"><strong>End Date:</strong> 2020-08-22</span>
-          <a href="/single-blog/2">More Details</a>
-        </div>
-      </article>
-    </li> -->
-
-  </ol>
-
+<div class="home">
```

Comment:
check out the formatting for this file

Reply:
Believe I fixed any formatting issues, let me know if not.